frontend/src/metabase/query_builder/containers/QueryBuilder.jsx

```diff
@@ -22,70 +22,73 @@ import { getIsNavbarOpen } from "metabase/selectors/app";
 import { getMetadata } from "metabase/selectors/metadata";
 import { getSetting } from "metabase/selectors/settings";
 import {
+  canManageSubscriptions,
   getUser,
   getUserIsAdmin,
-  canManageSubscriptions,
 } from "metabase/selectors/user";
 
 import * as actions from "../actions";
 import View from "../components/view/View";
 import { VISUALIZATION_SLOW_TIMEOUT } from "../constants";
 import {
+  getAutocompleteResultsFn,
```

Comment:
Looks like this got alphabetized automatically. I think we all know I wouldn't do this manually :joy: